.gitignore

```diff
@@ -15,3 +15,25 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+
+.idea/inspectionProfiles/Project_Default.xml
+
+.idea/inspectionProfiles/profiles_settings.xml
+
+.idea/jsLibraryMappings.xml
+
+.idea/jsLinters/jshint.xml
+
+.idea/libraries/bower_components_linkie_webapp.xml
+
+.idea/libraries/node_modules_linkie_webapp.xml
+
+.idea/linkie-webapp.iml
+
+.idea/misc.xml
+
+.idea/modules.xml
+
+.idea/vcs.xml
+
+.idea/workspace.xml
```

Comment:
Good .idea ^^

2016-04-25 10:09 GMT+02:00 tmakuch <notifications@github.com>:

> In .gitignore
> <https://github.com/Divoolej/linkie-webapp/pull/2#discussion_r60874861>:
>
> > +
> > +.idea/jsLinters/jshint.xml
> > +
> > +.idea/libraries/bower_components_linkie_webapp.xml
> > +
> > +.idea/libraries/node_modules_linkie_webapp.xml
> > +
> > +.idea/linkie-webapp.iml
> > +
> > +.idea/misc.xml
> > +
> > +.idea/modules.xml
> > +
> > +.idea/vcs.xml
> > +
> > +.idea/workspace.xml
>
> Ignore .idea instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Divoolej/linkie-webapp/pull/2/files/f94ebdb4e576cc4e3e42713db033eef116dbaa31#r60874861>
>